gwf/cli/arg_parsing.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import, print_function
+from ..utils import cache, import_object
+from ..core import PreparedWorkflow
+import argparse
+
+
+class SubCommand(object):
+    """Super-class for sub-commands."""
+
+    # Class variable used to get the active workflow.
+    _workflow = None
+    @property
+    @cache
+    def workflow(cls):
+        if cls._workflow is None: # pragma: no cover
```

Comment:
As you write, no reason to check this. We'll make sure to always set the workflow.